index.html

```diff
@@ -5,21 +5,21 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<link rel="stylesheet" type="text/css" href="assets/css/paper.css">  
 	<link rel="icon" href="assets/images/color-circle.png">
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
We are using only a handful of icons. So instead of using font awesome, use SVG icons. Using font awesome increases the load time of the website.